### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Recompare pop up_AI4D: Control type of the "Re Compare" Popup is set as "Tree"

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6664]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

Additional Details : 
Tools: AI4D
MAS reference-MAS 4.1.2, MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Ok button & hit enter.
11. Navigate till Options menu from Schema compare Menu bar & hit enter.
12. Check or uncheck some options & hit Ok.
13. Validate Recompare pop will open
14. Open AI4D and check the control type of "Re Compare popup".

**Actual:**
Control type of the "Re Compare" Popup is not correct., It is present as Tree and the content of the popup is also showing as "Tree item".

**Expected:**
Control type of the Popup should be provided as "Re Compare" dialog and the content should not be implemented as Tree items.
 
**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader user will not be able to know the information about the popup.

**MAS Reference:**
MAS4.1.2- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

MAS1.3.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
![6664_ Accessibility A11y_AzureDataStudio(Schema,Dacpac,Backup)_Recompare pop up_AI4D](https://user-images.githubusercontent.com/24580570/62778853-33b4e300-bacf-11e9-9f28-97c1b9697a6a.png)

**Comment by udeeshagautam** [comment]
@fsteffi  ADS uses the same control for toast notification as VSCode. Opened an issue in VSCode Repo. 
https://github.com/microsoft/vscode/issues/82728

@hannahqin , can you please do required for tracking this as external bug
